src/api_authentication_app_skeleton/src/actions/mixins/api/auth/helpers.cr

```diff
@@ -0,0 +1,27 @@
+module Api::Auth::Helpers
+  def current_user? : User?
+    auth_token.try do |value|
+      user_from_auth_token(value)
+    end
+  end
+
+  private def auth_token : String?
+    bearer_token || token_param
+  end
+
+  private def bearer_token : String?
+    context.request.headers["Authorization"]?
+      .try(&.gsub("Bearer", ""))
```

Comment:
Works if using `Bearer` or not

Reply:
We just need to make sure we add guides that show how to update this. (i.e. you don't want param tokens, and require the word "Bearer" to be present in the header, etc...)

Reply:
Yeah great idea